src/datahandlers/efo.py

```diff
@@ -0,0 +1,116 @@
+from src.prefixes import EFO,ORPHANET
+from src.babel_utils import pull_via_urllib
+from src.babel_utils import make_local_name
+from src.util import Text
+import pyoxigraph
+
+
+
+def pull_efo():
+    _=pull_via_urllib('http://www.ebi.ac.uk/efo/','efo.owl', subpath='EFO', decompress=False)
```

Comment:
This is fascinating. What's the benefit of assigning it to underscore?

Reply:
Ah, I have seen that used in python specifically to denote that there is a value returned, but you're not going to do anything with it.  I don't use it consistently, for better or worse.

Reply:
e.g. https://towardsdatascience.com/5-different-meanings-of-underscore-in-python-3fafa6cd0379.